### ISSUE TITLE
Checks run on all files in the meta directory

### ISSUE THREAD

**Issue by robertdebock** [ansible-community/ansible-lint#571]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible --version
ansible 2.8.2
  config file = None
  configured module search path = [u'/home/robertdb/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.16 (default, Apr 30 2019, 15:54:43) [GCC 9.0.1 20190312 (Red Hat 9.0.1-0.10)]
```

```
ansible-lint --version
ansible-lint 4.1.0
```

- ansible installation method: pip
- ansible-lint installation method: pip

# Desired Behaviour
Linting happens on `meta/main.yml` only.

# Actual Behaviour (Bug report only)
Linting happens on any .yml file in the `meta` directory.

In my case I've added a `meta/version.yml` file, which does not meet any of the linting requirements, but is being checked.

Please have a look at the [error](https://travis-ci.org/robertdebock/ansible-role-tomcat/jobs/572728986) and the [code](https://github.com/robertdebock/ansible-role-tomcat/tree/master/meta)

**Comment by robertdebock** [comment]
I've worked around the situation (aka "for others who hit this issue"):

molecule.yml:
```yaml
provisioner:
  name: ansible
  lint:
    name: ansible-lint
    options:
      exclude:
        - meta/version.yml
```

Still I think it's unexpected that all files in `meta/` are evaluated.

**Comment by ssbarnea** [comment]
That is no longer true. The linter can lint any files but keep in mind that each file from within meta can have different structure, they are not created equal.
